index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Portfolio Trial</title>
+<link rel="stylesheet" href="style.css"/>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap">
+</head>
+<body>
+<section class = "toolbar-section">
+<div class="font">
+    <div class="container-1">
+        <div class="font">
+            <header class="header">
+                <a class="myLogo"> My Logo </a>
+                <nav class="burger">
+                    <ul class="ul-image">
+                        <li>
+                            <a href="#">
+                <img src="/Images/Union.png" alt="union-background"/>
+            </a>
```

Comment:
- [ ] Great job so far on working on the styles but as the previous code reviewer asked to adjust the style so I can see here that the logo and the Hamburger menu are not aligned with the Figma design so please Fix the aligning and use list-style: none to delete the bullet points.

   - Your design :
       
![image](https://user-images.githubusercontent.com/58553711/176684635-729d6012-05f0-4e37-8eae-0ae818add6a1.png)

   - Figma design 
 
![image](https://user-images.githubusercontent.com/58553711/176686445-3d975a7a-44fb-4a4f-97ff-96f5766ebf74.png)

Reply:
- [ ] [optional] @peterboro great work implementing changes requested, you could also improve in your design check the comparison below some background images are not well positioned.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/176729265-0fa28e27-f246-415e-9b6c-59239fa026d4.png) | ![image](https://user-images.githubusercontent.com/61408860/176729330-630cbf53-360f-4388-afd4-3238b712da1f.png)